zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+if(CONFIG_WOLFTPM)
```

Comment:
Please add a GitHub CI action to test this. Use https://github.com/wolfSSL/wolfssl/blob/master/.github/workflows/zephyr.yml as template.